src/App.js

```diff
@@ -1,25 +1,11 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <h1>Math Magicians</h1>
+      <Calculator />
     </div>
```

Comment:
- [ ] Your `App.js` component should be class-based, just like you did with the calculator and button components

Reply:
Addressed this